README.md

```diff
@@ -100,137 +61,95 @@ After you're finished please remove all the comments and instructions!
 
 ### Key Features <a name="key-features"></a>
 
-> Describe between 1-3 key features of the application.
-
-- **[key_feature_1]**
-- **[key_feature_2]**
-- **[key_feature_3]**
+- **[Create database]**
+- **[Create table]**
+- **[Insert into database]**
+- **[Query database]**
+- **[Update and delete from database]**
+- **[SQl transactions]**
+- **[SQl Aggregates]**
+- **[Alter table attributes and constraints]**
+- **[SQL JOIN]**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- LIVE DEMO -->
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-> Add a link to your deployed project.
-
-- [Live Demo Link](<replace-with-your-deployment-URL>)
+> N/A
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- GETTING STARTED -->
 
 ## 💻 Getting Started <a name="getting-started"></a>
 
-> Describe how a new developer could make use of your project.
-
 To get a local copy up and running, follow these steps.
 
 ### Prerequisites
 
 In order to run this project you need:
+Install Postgres SQL
 
-<!--
-Example command:
-
-```sh
- gem install rails
-```
- -->
+> [Postgress SQL download](https://www.postgresql.org/download/)
 
 ### Setup
 
 Clone this repository to your desired folder:
 
-<!--
 Example commands:
 
 ```sh
-  cd my-folder
-  git clone git@github.com:myaccount/my-project.git
+  git clone git@github.com:Kel-nana/vet_clinic_project.git
+  cd vet_clinic
 ```
---->
-
-### Install
-
-Install this project with:
-
-<!--
-Example command:
-
-```sh
-  cd my-project
-  gem install
-```
---->
 
 ### Usage
 
 To run the project, execute the following command:
 
-<!--
-Example command:
-
 ```sh
-  rails server
+  psql -U postgres
 ```
---->
 
-### Run tests
+> Enter your password and continue with sql commands.
 
-To run tests, run the following command:
+### Run tests
 
-<!--
-Example command:
-
-```sh
-  bin/rails test test/models/article_test.rb
-```
---->
+To run tests, run the following command: N/A
 
 ### Deployment
 
 You can deploy this project using:
 
-<!--
-Example:
+<!-
 
 ```sh
+N/A
 
 ```
- -->
+
+-->
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- AUTHORS -->
 
-## 👥 Authors <a name="authors"></a>
+## 👥 Author : <a name="authors">Kelvin Irungu</a>
 
-> Mention all of the collaborators of this project.
-
-👤 **Author1**
-
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
-
-👤 **Author2**
-
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+- GitHub: [@Kel-nana](https://github.com/Kel-nana)
+- Linkedin: [@kelvin-irungu](https://www.linkedin.com/in/kelvin-irungu-838923249/)
+- Twitter: [@kelvin-irungu](https://twitter.com/home)
```

Comment:
- [ ] To make your documentation looks more professional and as required for all projects, I think it would better if  you add all the project contributors in the `Authors` section.